fastlane/.gitignore

```diff
@@ -2,6 +2,14 @@
 /report.xml
 /screenshots/*/*.png
 /screenshots/screenshots.html
```

Comment:
Do we still need these?

Reply:
Yes. We dont want to track `screenshots.html` which is not excluded in any other ignore line.

Reply:
I dont think so. Its just legacy ingore copied from fastlane